scripts/add-at-macro/src/add_at_macro/core.clj

```diff
@@ -0,0 +1,374 @@
+(ns add-at-macro.core
+  (:require [clojure.java.io :as io]
+            [rewrite-clj.zip :as z]
+            [clojure.set :as clj-set])
+  (:import (java.io File FileNotFoundException)))
+
+(defn get-alias
+  "Gets re-com alias in require vector i.e `[re-com.core ... :as rc]
+  returns the string rc. This is used to find re-com usages in a file."
+  ([loc]
+   (let [alias (if-let [as-kw (z/find-value loc z/next ':as)]
+                 (-> as-kw z/next z/string))]
+     alias)))
+
+(defn find-at-loc
+  "Takes a loc in the form of a vector containing re-com components i.e [p h-box at ...]
+  and returns the loc if `at` macro is in the vector."
+  [loc]
+  (loop [loc (z/down loc)]
+    (cond
+      (z/end? loc) nil
+      (-> loc z/string (= "at")) loc
+      :else
+      (recur (z/right loc)))))
+
+(defn find-require-macros-in-require
+  "Finds `:refer-macros` in a re-com import form which is in the `:require` form, i.e
+  (:require [re-com.core :refer-macros [] ...] ...). If found, we search if the `:refer-macros`
+  option loads `at` macro and we remove it if true. The at-macro will be added to the `:refer`
+  option."
+  [loc]
+  (let [refer-macros  (z/find-value loc z/next ':refer-macros)
+        refer-macros  (z/right refer-macros)
+        at-macro      (when refer-macros
+                        (find-at-loc refer-macros))]
+    (if at-macro
+      (-> at-macro z/remove z/up z/up)
+      loc)))
+
+(defn append-at-to-recom-require-form
+  "Checks if re-com import statement has :refer option, appends 'at' macro if True, and
+  adds refer option with 'at' macro if not. Loc is [re-com.core ...]"
```

Comment:
I would structure your docstrings using the form 

> Given `<argument desciption>`, returns `<return value description>`

and then move on to some clarifying examples.  Perhaps like this sketch ... 

---
Given `loc` which is require vector such as `[XXXX]`, will return a modified vector `[XXXX]`

It might also transform `loc` from: 
     [re-com.core :as rc :refer [XXX]]
to: 
     [re-com.core :as rc :refer [XXX]]
   
And, also, a `loc` from: 
    blah 
to 
    blah